lib/WebDriverExpectedCondition.php

```diff
@@ -583,4 +584,65 @@ function (WebDriver $driver) use ($condition) {
             }
         );
     }
+
+    /**
+     * An expectation with the logical AND condition of the given conditions.
+     *
+     * @param WebDriverExpectedCondition[] $conditions Array of the conditions to be satisfied.
+     * @return static|false Array of the getApply() values of all satisfied conditions or FALSE if some condition
```

Comment:
```suggestion
     * @return static array of the getApply() values of all satisfied conditions or FALSE if some condition
```
I am sorry, I misunderstood the code
There is no return false for the function but only a false value in the object
Could you make it clearer to understand ?